config/sync/core.extension.yml

```diff
@@ -39,6 +40,10 @@ module:
   image: 0
   image_style_warmer: 0
   inline_form_errors: 0
+  jquery_ui: 0
+  jquery_ui_datepicker: 0
+  jquery_ui_slider: 0
+  jquery_ui_touch_punch: 0
```

Comment:
These modules are dependencies of better_exposed_filters, so have to be enabled.
jquery_ui is admittedly pretty old, but it's still being used in certain places in Drupal. 
Jquery itself is being phased out of Drupal core, and there is a plan to remove it from the better_exposed_filters module (https://www.drupal.org/project/better_exposed_filters/issues/3210944) but it will probably take a while.